workbench-core/authorization/src/dynamicAuthorization/wbcGroupManagementPlugin.ts

```diff
@@ -242,5 +249,29 @@ export class WBCGroupManagementPlugin implements GroupManagementPlugin {
       }
       throw error;
     }
+    return { data: { status: newStatus } };
```

Comment:
its odd to see the return after the catch block, can we move it to the end of the try block?